output/generated/strings/Basic_Emoji-negative-P.js

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 Mathias Bynens. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Mathias Bynens
+description: >
+  Negating Unicode property escapes for `Basic_Emoji` (property of strings) with `\P{…}` throws an early error.
+info: |
+  Generated by https://github.com/mathiasbynens/unicode-property-escapes-tests
+  Unicode v14.0.0
+esid: sec-isvalidregularexpressionliteral
+features: [regexp-unicode-property-escapes, regexp-v-flag]
```

Comment:
Is the new feature tag `regexp-v-flag` okay? It would contain not just these tests (specific to `\p{PropertyOfStrings}`) but also the set notation syntax and semantics: https://github.com/tc39/proposal-regexp-set-notation But those tests would be added separately, and manually written.